src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -646,10 +652,11 @@ public virtual ParamIdentifierStatus GetIdentifierParamStatus(Features features,
         /// <summary>
         /// Returns whether the parameter can be represented by an identifier.
         /// </summary>
+        /// <param name="node">TexlNode. Used by functions that dont have args based on order e.g. Summarize.</param>
         /// <param name="features">The features enabled for the expression.</param>
         /// <param name="index">Parameter's index.</param>
         /// <returns>true if the parameter can be an identifier, false otherwise.</returns>
-        public bool ParameterCanBeIdentifier(Features features, int index)
+        public virtual bool ParameterCanBeIdentifier(TexlNode node, int index, Features features)

```

Comment:
>virtual [](http://example.com/codeflow?start=15&length=7)

We already have one virtual method in TexlFunction about identifier status for parameters (the one above that). Ideally we should not have multiple virtuals for the same scenario. Can we update the GetIdentifierParamStatus to receive the TexlNode, and not make this one virtual?